src/Symfony/Component/Console/Command/Command.php

```diff
@@ -605,6 +605,10 @@ public function getUsages()
      */
     public function getHelper($name)
     {
+        if (null === $this->helperSet) {
+            throw new LogicException(sprintf('Unable to retrieve helper "%s" because there is no HelperSet defined for this command. Did you forget to call %s::setHelperSet()?', $name, get_class($this)));
```

Comment:
The message can be confusing as you usually do not set the helper set directly on the command, but configure it for the application instead.